README.md

```diff
@@ -0,0 +1,83 @@
+# STOCKIST WEB APP - REACT CAPSTONE PROJECT
+
+The project, a frontend web application, catalogue of financial values using [Stock API](https://financialmodelingprep.com/developer/docs/)  and collect the data from my React Application.
+
+The application features includes:
+- Listing of Most Active Stocks
+- Listing of Most Gainer's Stocks
+- Listing of Most Loser's Stocks
+- Filter between those 3, most active, gainers and losers
+- Open individual stock item
+- Search for a particular stock item, example 'AAPL'
+- Listing of the lastest stock news
+- Dynamic routes for individual stock item, example `/stocks/AAPL`
+
+![screenshot](./stockist.gif)
+
+## Live Demo
+[Live Demo](https://capstone-finance-rebel216.herokuapp.com/Capstone-React-Finance/)
```

Comment:
- [ ] the link that you provided is not working please make sure to provide the working link to your deployed version of your app 
<img width="1229" alt="Screen Shot 2022-10-08 at 10 35 38" src="https://user-images.githubusercontent.com/61790587/194700909-dbe8ab6e-7d1f-4b1b-8d95-ea8658f99c6e.png">